.github/workflows/android-perf.yml

```diff
@@ -0,0 +1,179 @@
+name: android-perf
+
+on:
+  schedule:
+    - cron: 0 0 * * *
+  pull_request:
```

Comment:
Note, pls remember to remove `pull_request` here before committing.  Otherwise, this runs on all PRs

Reply:
Yeah, it's for testing purpose only